### PR TITLE
Updated with slit

### PR DESCRIPTION
**Pull request by JayChittidi**
Added rectangular slit via photuils that can be added to an image. Also commented out flipping RA (it was already flipped if using updated astropy)

### CODE REVIEW

File: frb/figures/finder.py [hunk 1]
```diff
@@ -109,8 +114,7 @@ def generate(image, wcs, title, log_stretch=False,
     cimg = ax.imshow(image, cmap='Greys', norm=norm)
 
     # Flip so RA increases to the left
-    ax.invert_xaxis()
-
+    #ax.invert_xaxis()
```
Comment (profxj) [inline frb/figures/finder.py]: Please make this option, e.g. `invert`.  I have a feeling we will want to 
use it when we make one in the Southern hemisphere.

File: frb/figures/finder.py [hunk 2]
```diff
@@ -19,6 +19,8 @@
 from astropy.coordinates import ICRS
 from astropy.time import Time
 
+from photutils import SkyRectangularAperture
```
Comment (profxj) [inline frb/figures/finder.py]: This adds a dependency to FRB.  I'm ok with that, but
I think we should make it optional.  And we should add
a note about it in the Install.  Here, wrap it in a `try/except`, 

```
try:
     from photutils import SkyRectangularAperture
except ImportError:
    flag_photu = False
else:
    flag_photu = True
```

and then only allow the `slit` option if `flag_photu=True`.

File: frb/figures/finder.py [hunk 3]
```diff
@@ -168,15 +172,26 @@ def generate(image, wcs, title, log_stretch=False,
...
-        ax.add_patch(r)
-    '''
+        # List of values - [coodinates, width, length, PA],
+        # e.g. [SkyCoords('21h44m25.255s',-40d54m00.1s', frame='icrs'), 1*u.arcsec, 10*u.arcsec, 20*u.deg]
+        slit_coords, width, length, pa = slit
+        
+        pa_deg = pa.to('deg').value
+        pa_rad = pa_deg * np.pi / 180.
+        
+        aper = SkyRectangularAperture(positions=slit_coords, w=width, h=length, theta=pa)
```
Comment (profxj) [inline frb/figures/finder.py]: should this be using `pa_rad`?
if not, do we need it?

File: frb/figures/finder.py (hunk 2 above)
Comment (JayChittidi) [reply frb/figures/finder.py]: Added changes you requested. Please see the new commit and let me know if there are additional changes that need to be made.

File: frb/figures/finder.py [hunk 4]
```diff
@@ -190,7 +198,9 @@ def generate(image, wcs, title, log_stretch=False,
         
         plt.text(0.5, -0.05, 'Slit PA={} deg'.format(pa_deg), color='purple',
                  fontsize=15, ha='center', va='top', transform=ax.transAxes)
-        
+    
+    if ((slit is not None) and (flag_photu is False)):
+        print('Slit cannot be placed with photutils package')
```
Comment (profxj) [inline frb/figures/finder.py]: Turn this into a `raise IOError` and have it say
that the slit option requires that photutils be installed.

**Comment by profxj** [comment]
One more fix @JayChittidi and also
we need to add notes on `photutils` in 
the `installing.rst` file in docs/

**Comment by ntejos** [comment]
Have fixed an issue with the slit PA. Please see last commit.

**Comment by profxj** [comment]
Thanks for the edits @ntejos 
@JayChittidi -- still need your final pass of edits then we can merge.  Thanks.

File: frb/figures/finder.py (hunk 4 above)
Comment (profxj) [reply frb/figures/finder.py]: @JayChittidi -- please get this one.  thanks.

Comment (JayChittidi) [reply frb/figures/finder.py]: Hi X, the commit from yesterday has this change.

**Comment by profxj** [comment]
merging

**Pull request merged**